### ISSUE TITLE
How do you assign an action to an LED for a specific mode (CM3 Throttle)

### ISSUE THREAD

**Issue by davidginzbourg** [charliefoxtwo/ViLA#10]
Hi, I'm not sure exactly how can an LED be assigned with a color in a specific mode, i.e. when the mode dial is on a certain mode.
I see there isn't a way to set logical operators so I'm guessing I can't do an AND with the mode dial.

**Comment by charliefoxtwo** [comment]
Correct, I don't currently have logical operators implemented (though they are planned).

That said, I _also_ don't currently have a way of monitoring the position of switches (like the mode dial) on the controllers. That would have to be a separate plugin.

You can use the VPC Software Suite if you just want to change the colors of the LEDs based on the mode dial. If you wanted to change the LEDs based on in-game events _and_ the mode dial, I don't have a solution for that currently. It's a good idea though!

**Comment by davidginzbourg** [comment]
> If you wanted to change the LEDs based on in-game events and the mode dial, I don't have a solution for that currently
Yep, that was what I was asking about
